### PR TITLE
Update ReviewVirtualSwapSettlement

### PR DESCRIPTION
**Pull request by nkrulikovsky**
## What does this PR do?

- Remove old `Button` and its `scss` completely
- Update `ReviewVirutalSwapSettlement`
- Remove `ReviewSwap.scss`

## Context

https://www.figma.com/file/Aa3IfDUO68aCNr8nZTuQ8S/Product-Design-with-MUI?node-id=257%3A31934

## Screenshot
Screen Shot 2022-04-14 at 1 51 20 AM
Screen Shot 2022-04-14 at 1 51 35 AM
Screen Shot 2022-04-14 at 1 51 52 AM
Screen Shot 2022-04-14 at 1 52 04 AM

### CODE REVIEW

**Comment by ug02fast** [comment]
Is it possible to align this arrow in the middle between the icons? 
image

**Comment by nkrulikovsky** [comment]
@ug02fast 

Thank you for reviewing this PR.
I aligned tokenIcon and arrow icon by giving them same size.

File: public/locales/en.json [hunk 1]
```diff
@@ -154,6 +155,8 @@
   "reviewMigration": "Review Migration",
   "reviewSwap": "Review Swap",
   "reviewWithdraw": "Review Withdraw",
+  "step2ReviewSettle": "Step 2: Review Settle",
+  "step2Withdraw": "Step 2: Review Withdraw",
```
Comment (ug02fast) [inline public/locales/en.json:159]:
```suggestion
  "step2ReviewWithdraw": "Step 2: Review Withdraw",
```

File: src/components/ReviewVirtualSwapSettlement.tsx [hunk 2]
```diff
@@ -73,107 +71,122 @@ function ReviewVirtualSwapSettlement({
...
-        <div className="from">
-          <TokenIcon symbol={fromToken.symbol} alt="icon" />
-          <span className="tokenName">{data.from.symbol}</span>
-          <div className="floatRight">
-            <span>{commify(data.from.value)}</span>
-          </div>
-        </div>
+    <div>
+      <Typography variant="h1" mb={3}>
+        {isWithdrawAction ? t("step2Withdraw") : t("step2ReviewSettle")}
```
Comment (ug02fast) [inline src/components/ReviewVirtualSwapSettlement.tsx:78]:
```suggestion
        {isWithdrawAction ? t("step2ReviewWithdraw") : t("step2ReviewSettle")}
```

**Review by ug02fast — CHANGES_REQUESTED** [review]
looks good just small suggestion

File: public/locales/en.json (hunk 1 above)
Comment (nkrulikovsky) [reply public/locales/en.json]: Updated!

File: src/components/ReviewVirtualSwapSettlement.tsx (hunk 2 above)
Comment (nkrulikovsky) [reply src/components/ReviewVirtualSwapSettlement.tsx]: Done!

**Review by ug02fast — APPROVED** [review]

**Pull request merged**